src/components/Cast/TextareaInfo.tsx

```diff
@@ -71,6 +75,11 @@ export default function ({
           </AccentText>
         </div>
       )}
+      {!!error && (
+        <div className={displayTo('md')}>
```

Comment:
Why are you using `displayTo('md')` here?

Reply:
To show small text errors on small screens.